tests/views/test_reporting_units.py

```diff
@@ -58,6 +58,8 @@
 
 with open(f"{project_root}/test_data/party/business_reporting_unit.json") as fp:
     business_reporting_unit = json.load(fp)
+with open(f"{project_root}/test_data/party/business_reporting_unit_no_enrolments.json") as fp:
```

Comment:
This config is for the scenario this PR is resolving. There are no respondents registered for the RU/survey and as such the hyperlink was malformed. This test is to confirm the fix.